app/content/serializers/registration.py

```diff
@@ -18,11 +20,11 @@ class Meta:
             "has_attended",
             "allow_photo",
             "created_at",
+            "submissions",
```

Comment:
Kan vel ikke være mer enn 1 submission per registration? Isåfall burde også navnet på feltet endres til entall: `submission`

Reply:
Tror det her er snakk om for evalueringer og påmeldingsskjemaer. Kan forenkles som du sier ved å dele de opp

Reply:
Kanskje kalle det survey_submission, så det er helt tydelig hva det er snakk om.